### PR TITLE
polishing of the tuning guide

### PR DESCRIPTION
**Pull request by Imaanpreet**
…igurations

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8, orcharhino 6.1 on EL7, orcharhino 6.2 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

**Comment by maximiliankolb** [comment]
Please rebase your branch to master and resolve the merge conflict.

**Comment by Imaanpreet** [comment]
@maximiliankolb Hello! could you please check now?

File: guides/common/modules/con_top-performance-considerations.adoc [hunk 1]
```diff
@@ -1,28 +0,0 @@
-[id="Top_Performance_Considerations_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/con_top-performance-considerations.adoc:1]: You've deleted the hole `guides/common/modules/con_top-performance-considerations.adoc` file. If this was intentional, you'd also have to remove the line that includes this file which you can find using `git grep "con_top-performance-considerations.adoc"` and remove all the links to it's anchor.

File: guides/common/modules/proc_benchmarking-raw-db-performance.adoc [hunk 2]
```diff
@@ -5,7 +5,7 @@ ifndef::orcharhino[]
 To get a list of the top table sizes in disk space for both Candlepin, Foreman, and Pulp check https://github.com/RedHatSatellite/satellite-support/blob/master/postgres-size-report[postgres-size-report] script in https://github.com/RedHatSatellite/satellite-support[satellite-support] git repository.
 endif::[]
 
-PGbench utility (note you may need to resize PostgreSQL data directory `{postgresql-lib-dir}` directory to 100 GiB or what does benchmark take to run) might be used to measure PostgreSQL performance on your system.
+PGbench utility (note you may need to resize PostgreSQL data directory `/var/lib/pgsql` directory to 100 GiB or what does benchmark take to run) might be used to measure PostgreSQL performance on your system.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_benchmarking-raw-db-performance.adoc:8]: I have introduced attributes for various directories in Foreman server. If you want to change it's values, please have a look at https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L79

File: guides/common/modules/proc_determining-hardware-and-os-configuration.adoc [hunk 3]
```diff
@@ -1,6 +1,5 @@
-[id="Configuring_Project_Environment_for_Performance_{context}"]
-= Configuring {Project} Environment for Performance
-
+[id="Determining_Hardware_and_OS_Configuration]
+= Determining Hardware and OS Configuration
```
Comment (maximiliankolb) [inline guides/common/modules/proc_determining-hardware-and-os-configuration.adoc:2]:
```suggestion
= Determining Hardware and OS Configuration

```

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

File: guides/common/modules/con_top-performance-considerations.adoc (hunk 1 above)
Comment (Imaanpreet) [reply guides/common/modules/con_top-performance-considerations.adoc:1]: Hello, this file only in one place as of now and I have removed it from master.

**Comment by Imaanpreet** [comment]
@maximiliankolb : Hello, can you please review it once again and let me know what are your thoughts?

File: guides/common/modules/proc_benchmarking-raw-db-performance.adoc [hunk 4]
```diff
@@ -5,7 +5,7 @@ ifndef::orcharhino[]
 To get a list of the top table sizes in disk space for both Candlepin, Foreman, and Pulp check https://github.com/RedHatSatellite/satellite-support/blob/master/postgres-size-report[postgres-size-report] script in https://github.com/RedHatSatellite/satellite-support[satellite-support] git repository.
 endif::[]
 
-PGbench utility (note you may need to resize PostgreSQL data directory `{postgresql-lib-dir}` directory to 100 GiB or what does benchmark take to run) might be used to measure PostgreSQL performance on your system.
+PGbench utility (note you may need to resize PostgreSQL data directory `postgresql-lib-dir` directory to 100 GiB or what does benchmark take to run) might be used to measure PostgreSQL performance on your system.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_benchmarking-raw-db-performance.adoc:8]:
```suggestion
PGbench utility (note you may need to resize PostgreSQL data directory `{postgresql-lib-dir}` directory to 100 GiB or what does benchmark take to run) might be used to measure PostgreSQL performance on your system.
```

File: guides/common/modules/proc_determining-hardware-and-os-configuration.adoc [hunk 5]
```diff
@@ -1,6 +1,5 @@
...
 CPU::
```
Comment (maximiliankolb) [inline guides/common/modules/proc_determining-hardware-and-os-configuration.adoc:3]:
```suggestion

CPU::
```

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
made two small comments.

**Comment by Imaanpreet** [comment]
> made two small comments.

Thank you.

File: guides/common/modules/proc_determining-hardware-and-os-configuration.adoc [hunk 6]
```diff
@@ -1,5 +1,5 @@
-[id="Configuring_Project_Environment_for_Performance_{context}"]
-= Configuring {Project} Environment for Performance
+[id="Determining_Hardware_and_OS_Configuration]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_determining-hardware-and-os-configuration.adoc:1]:
```suggestion
[id="Determining_Hardware_and_OS_Configuration_{context}"]
```

**Comment by maximiliankolb** [comment]
Is there a reason why you remove the top performance considerations?

**Comment by Imaanpreet** [comment]
> Is there a reason why you remove the top performance considerations?

Hello, we have added this chapter when we were having tuning guide on google document & PDF that points us to section which has the exact information. 

Now, we have the tuning guide with Foreman & Red Hat documentation which have all the links with proper information.

**Comment by Imaanpreet** [comment]
Hello @maximiliankolb : Please let me know if you have any questions/suggestions on this PR ? If no, can I have any update on this PR?

**Comment by ekohl** [comment]
Right now CI fails with:
```
asciidoctor: ERROR: master.adoc: line 19: include file not found: /home/runner/work/foreman-documentation/foreman-documentation/guides/doc-Tuning_Performance/common/modules/proc_configuring-project-environment-for-performance.adoc
```

This is because you renamed the file, but didn't update the `include` statement(s).

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
On thread is still open for now.

**Review by maximiliankolb — APPROVED** [review]
LGTM.

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   21ccc9d64..3b35fd39b  3.6 -> 3.6
   cebf594f7..34d2053c9  3.5 -> 3.5
   df14c375f..13eb367af  3.4 -> 3.4
   9ca21d9bf..d8717e65a  3.3 -> 3.3

**Pull request merged**